lenskit-cli/src/main/java/org/lenskit/cli/commands/GlobalRecommend.java

```diff
@@ -25,16 +25,17 @@
 import net.sourceforge.argparse4j.impl.Arguments;
```

Comment:
Ehh, that makes some things super-weird.

We should come up with a better name, but perhaps that could be a separate ticket.